lib/mermaid.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { mermaidAPI } from 'mermaid/dist/mermaid.js'
```

Comment:
mermaid.mermaidAPI.render is [marked as deprecated](https://github.com/mermaid-js/mermaid/blob/60280361b0c89a96f6d377e34f7f5cd32b52c775/packages/mermaid/src/mermaidAPI.ts#L328)